scripts/upload_sample.py

```diff
@@ -44,36 +48,40 @@ class ExecutionContext(BaseModel):
     token: TokenObject | None = None
 
 
+def _rel_to_abs_path(path: Path | str, base_path: Path) -> Path:
```

Comment:
Looks like something that could be placed in a low-level util module?

What is the purpose of the "_" prefix here, to show it is module private? (Not a bad idea, just haven't seen it elsewhere in the codebase here yet)

Reply:
I plan to move the upload functionality to PRP in the near future so I keep as is for now.

Yeah the "_" prefix is to show that the function is intended to be private.